src/main/flight/pid.h

```diff
@@ -198,3 +199,13 @@ bool pidOsdAntiGravityActive(void);
 bool pidOsdAntiGravityMode(void);
 void pidSetAntiGravityState(bool newState);
 bool pidAntiGravityEnabled(void);
+
+#ifdef UNIT_TEST
+extern float axisError[XYZ_AXIS_COUNT];
+void applyItermRelax(const int axis, const float ITerm,
+    const float gyroRate, float *itermErrorRate, float *currentPidSetpoint);
+void applyAbsoluteControl(const int axis, const float gyroRate, const bool itermRelaxIsEnabled,
+    const float setpointLpf, const float setpointHpf,
+    float *currentPidSetpoint, float *itermErrorRate);
+void rotateITermAndAxisError();
+#endif
```

Comment:
Missing newline at end of file.

Reply:
Is there actually a problem with this?

Reply:
What? The missing newline? Just stylistic and trying to be consistent with @DieHertz 's reviews.

Reply:
Ok. I don't see this as a style issue that will result in maintenance problems down the line, so I'd be inclined to accept file endings both ways.

Reply:
GCC with Wall and Werror may generate and error for no newline at end of file

Reply:
Every file in the project ends with a new line. At least last time i checked for redundant and tailing whitespaces and stuff. I do not see why suddenly should not do it. It just will create a unneeded line change the next one saves it with c editor environment.

Beside the all known compiler warning  message "No newline at end of file"

Reply:
@wind0r: One reason to not worry about this is that there are many more important code quality issues to worry about - we have to pick our fights.